### ISSUE TITLE
can't create a "run all tests" configuration.

### ISSUE THREAD

**Issue by OneOfOne** [Microsoft/vscode-go#911]
I'm trying to do the equivalent of `go test -v ./...`, however everything I tried so far failed.

```
	{

		"name": "Launch test package",
		"type": "go",
		"request": "launch",
		"mode": "test",
		"program": "${workspaceRoot}/..."
	}
``` 
Fails with: `The program "xxxxxx ./..." does not exist: Error: ENOENT: no such file or directory, lstat 'xxxxxx ./...'`

```
	{
		"name": "Launch test package",
		"type": "go",
		"request": "launch",
		"mode": "test",
		"args": [
			"-v",
			"./..."
		],
		"program": "${workspaceRoot}"
	}
```

Fails with: 
```
can't load package: package xxx: no buildable Go source files in xxxxx
Process exiting with code: 1
```

**Comment by OneOfOne** [comment]
Note that I can accomplish this using:

```
	{
		"name": "Test All",
		"type": "go",
		"request": "launch",
		"mode": "exec",
		"program": "/home/oneofone/bin/go",
		"cwd": "${workspaceRoot}",
		"args": [ "test", "-v", "./..."]
	}
```

**Comment by ramya-rao-a** [comment]
Do you intend to debug these tests or just run them?

**Comment by OneOfOne** [comment]
@ramya-rao-a just run them.

**Comment by ramya-rao-a** [comment]
https://github.com/Microsoft/vscode-go/issues/911#issuecomment-291231328 actually uses delve and debugs through all the tests.

There is a `Go: Test Package` command that will run all the tests in the current package
But I am assuming yours is the case of multiple packages?

**Comment by OneOfOne** [comment]
Yep, I have pkg1/sub1 pkg1/sub2, normally I'd run go test ./... in the root of that pkg.

It might be helpful / handy to have an option to do that like `Go: Test All` or something.

**Comment by ramya-rao-a** [comment]
Sure, that's a valid feature request. Should be easy enough to do.

**Comment by ramya-rao-a** [comment]
The command `Go: Test All Packages in Workspace` is now available in the latest update (0.6.59)
